src/controllers/pages/index.js

```diff
@@ -24,4 +24,11 @@ export default class IndexController {
             hashHistory.push('/overdue');
         });
     }
+
+    static getStudents(){
```

Comment:
It's because we have to get the models from the server, otherwise we can't display the name of the items that the student has checked out.